revoked.md

```diff
@@ -0,0 +1,9 @@
+# Revoked Behaviors and Methods
```

Comment:
Another to add: C0013 UDP Communication because UDP + TCP were combined to become C0001 Socket Communication. Happened at v2.1 (so whatever date that was released).